pkg/ui/workspaces/cluster-ui/BUILD.bazel

```diff
@@ -2,6 +2,112 @@ load("@npm//webpack-cli:index.bzl", webpack = "webpack_cli")
 load("@build_bazel_rules_nodejs//:index.bzl", "js_library")
 load("@npm//@bazel/typescript:index.bzl", "ts_project")
 
+# do not include following packages:
+# "@npm//@cockroachlabs/crdb-protobuf-client",
+DEPENDENCIES = [
```

Comment:
It is possible to specify all "node_modules" dependencies with "@npm//:node_modules" but it our case I have to exclude particular NPM package (`@npm//@cockroachlabs/crdb-protobuf-client`) which is symlink to `//pkg/ui/workspaces/db-console/src/js:crdb-protobuf-client`). 
This list includes all dependencies from "package.json" file.

Reply:
Will the build fail if someone forgets to manually add something to this list? Just want to make sure this kind of thing doesn't go unnoticed.

Reply:
Correct, it will fail entire build.